src/main/resources/db/migration/0350/V367__FunderReports.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE funder.funder_reports (
+    report_id BIGINT PRIMARY KEY REFERENCES accelerator.reports,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    report_frequency_id INTEGER NOT NULL REFERENCES accelerator.report_frequencies,
+    report_quarter_id INTEGER REFERENCES accelerator.report_quarters,
+    start_date DATE NOT NULL,
+    end_date DATE NOT NULL CHECK (end_date > start_date),
+    highlights TEXT,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL
```

Comment:
Would it make sense to change these to `published_by` and `published_time` to make the semantics clearer?